### PR TITLE
dubbogo nacos registry sample

### PR DESCRIPTION
**Pull request by Patrick0308**

### CODE REVIEW

File: golang/registry/nacos/dubbo/go-server/profiles/dev/server.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  module : "dubbogo user-info server"
+  version : "0.0.1"
+  owner : "ZX"
+  environment : "dev"
+
+registries :
+  "nacos1":
+      protocol: "nacos"
+      timeout	: "3s"
+      address: "127.0.0.1:8848"
```
Comment (lzp0412) [inline golang/registry/nacos/dubbo/go-server/profiles/dev/server.yml]: suggest to use "console.nacos.io:80" which is a public nacos server address.

File: golang/registry/nacos/dubbo/java-server/src/main/resources/META-INF/spring/dubbo.provider.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<beans xmlns="http://www.springframework.org/schema/beans"
+	   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	   xmlns:dubbo="http://code.alibabatech.com/schema/dubbo"
+	   xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+	http://code.alibabatech.com/schema/dubbo http://code.alibabatech.com/schema/dubbo/dubbo.xsd">
+
+	<!-- 应用名 -->
+	<dubbo:application name="user-info-server"/>
+	<!-- 连接到哪个本地注册中心 -->
+	<dubbo:registry id="ikurento"  address="nacos://127.0.0.1:8848" />
```
Comment (lzp0412) [inline golang/registry/nacos/dubbo/java-server/src/main/resources/META-INF/spring/dubbo.provider.xml]: suggest to use "console.nacos.io:80" which is a public nacos server address.

File: golang/registry/nacos/dubbo/go-client/profiles/dev/client.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  module : "dubbogo user-info client"
+  version : "0.0.1"
+  owner : "ZX"
+  environment : "dev"
+
+registries :
+  "nacos1":
+    protocol: "nacos"
+    timeout: "3s"
+    address: "127.0.0.1:8848"
```
Comment (lzp0412) [inline golang/registry/nacos/dubbo/go-client/profiles/dev/client.yml]: suggest to use "console.nacos.io:80" which is a public nacos server address.

File: golang/registry/nacos/dubbo/java-client/src/main/resources/META-INF/spring/dubbo.consumer.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:dubbo="http://code.alibabatech.com/schema/dubbo"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+	http://code.alibabatech.com/schema/dubbo http://code.alibabatech.com/schema/dubbo/dubbo.xsd">
+
+
+	<!-- 消费方应用名，用于计算依赖关系，不是匹配条件，不要与提供方一样 -->
+	<dubbo:application name="user-info-client" />
+	<!-- 连接到哪个本地注册中心 -->
+	<dubbo:registry id="ikurento"  address="nacos://127.0.0.1:8848" />
```
Comment (lzp0412) [inline golang/registry/nacos/dubbo/java-client/src/main/resources/META-INF/spring/dubbo.consumer.xml]: suggest to use "console.nacos.io:80" which is a public nacos server address.

File: golang/registry/nacos/dubbo/go-server/profiles/dev/server.yml (hunk 1 above)
Comment (Patrick0308) [reply golang/registry/nacos/dubbo/go-server/profiles/dev/server.yml]: We can't ensure the stability of the `console.nacos.io:80` service.

**Pull request merged**